README.md

```diff
@@ -176,12 +176,13 @@ If you want to disable media control auto hide, add `hideMediaControl: false` in
 When embedded with width less than 320, volume bars are hidden. You can force this behavior for all sizes by adding `hideVolumeBar: true`.
 
 ##### Watermark
-Put `watermark: http://url/img.png` on your embed parameters to automatically add watermark on your video. Choose corner position by defining position parameter. Positions can be `bottom-left`, `bottom-right`, `top-left` and `top-right`. Example:
+Put `watermark: http://url/img.png` on your embed parameters to automatically add watermark on your video. Choose corner position by defining position parameter. Positions can be `bottom-left`, `bottom-right`, `top-left` and `top-right`. To define URL to open when the watermark is clicked use watermarkLink parameter. If watermarkLink parameter not defined watermark will not be clickable. Example:
```

Comment:
"To define an URL ... clicked, use `watermarkLink` parameter. If the watermarkLink is not defined, the watermark ...."